plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt

```diff
@@ -40,7 +40,21 @@ class CodeWhispererProjectStartupActivity : StartupActivity.DumbAware {
     override fun runActivity(project: Project) {
         if (!isQConnected(project)) return
 
-        // ---- Everything below will be triggered only when CW is enabled, everything above will be triggered once per project ----
+        // ---- Everything below will be triggered only when CW is enabled ----
+
+        val actionManager = CodeWhispererExplorerActionManager.getInstance()
+        val scanManager = CodeWhispererCodeScanManager.getInstance(project)
+        actionManager.setMonthlyQuotaForCodeScansExceeded(false)
+        //  Run Proactive Code File Scan and disabling Auto File Scan for Builder ID Users.
+        if (isUserBuilderId(project)) {
+            actionManager.setAutoCodeScan(project, false)
+        } else {
+            // Setting up listeners for Auto File Code Scan triggers.
+            scanManager.setEditorListeners()
+            scanManager.debouncedRunCodeScan(CodeWhispererConstants.CodeAnalysisScope.FILE)
+        }
+
+        // ---- Everything above will be triggered once per project and everything below will be triggered once after startup ----
```

Comment:
Nit: I feel like stating above + below in this comment makes it confusing, especially when you have a comment starting this block.

Reply:
Makes sense.. Just tried to delineate block with comments so it wouldn't be confusing.

Reply:
+1 to Bryce's comment, maybe could shorten this comment to `// trigggered after startup`. The everything above bit is redundant since it's already mentioned prior to this.